src/avatar.js

```diff
@@ -0,0 +1,78 @@
+import checkAccount from '~/common/checkAccount';
+import checkOptions from '~/common/checkOptions';
+
+/**
+ * Avatar submodule to upload and delete images from storage.
+ *
+ * @access private
+ *
+ * @param {Web3} web3 - Web3 instance
+ * @param {Object} utils - utils module instance
+ *
+ * @return {Object} - avatar module instance
+ */
+export default function createAvatarModule(web3, utils) {
+  return {
+    /**
+     * Upload an avatar image to storage.
+     *
+     * @namespace core.avatar.upload
+     *
+     * @param {Object} account - web3 account instance
+     * @param {Object} userOptions - options
+     * @param {object} userOptions.data - avatar image file
+     *
+     * @return {object} - Returns url, file name and file type of the uploaded image
+     */
+    upload: async (account, userOptions) => {
+      checkAccount(web3, account);
+
+      const options = checkOptions(userOptions, {
+        data: {
+          type: 'object',
+        },
+      });
+
+      const { data } = options;
+
+      return await utils.requestAPI({
```

Comment:
You can omit using in this method the "async/await" keywords as it will result in the same output